packages/shared-business/src/components/BeneficiaryForm.tsx

```diff
@@ -384,6 +384,28 @@ export const BeneficiaryForm = forwardRef<BeneficiaryFormRef | undefined, Props>
 
     const hasBeenSubmittedOnce = useRef(false);
 
+    useEffect(() => {
+      if (initialState != null) {
+        // validate all fields on mount to display errors on edition open
+        Promise.all([
+          validateField("firstName"),
```

Comment:
I think you can call `submitForm` instead and noop the result.

Reply:
yes I didn't have this idea, will try

Reply:
It works perfectly well thanks for this top. Just pushed 449eceb9a89367528a5e2375459a2bda2d6a1dad